lib/tool-bar-button-view.js

```diff
@@ -11,6 +11,13 @@ module.exports = class ToolBarButtonView {
     this.group = group;
     this.enabled = true;
 
+    if (options.color) {
```

Comment:
Minor thing & personal preference, but for maintainability I would suggest moving this to later in the function. E.g. after the [`options.text` if-statement](https://github.com/aminya/tool-bar/blob/09c85114a26cbcb7af78eb53a35253aa35e25139/lib/tool-bar-button-view.js#L62)? What do you think?